test/src/test-common.h

```diff
@@ -4,6 +4,14 @@
 #include "uv.h"
 #include "uv_link_t.h"
 
+#ifdef _WIN32
+#include <winsock2.h>
```

Comment:
Sorry, few more nits that I forgot to mention. Lets ident these includes with space between `#` and `include: `# include`.